### ISSUE TITLE
Cannot build Xamarin.IOS 

### ISSUE THREAD

**Issue by whann0205** [EdCharbeneau/BlazorPro.Spinkit#13]
Der Pfad "..\..\..\..\..\..\..\..\Users\hann.HANN-EDV\.nuget\packages\blazorpro.spinkit\1.2.0\contentFiles\any\netstandard2.1\libman.json" würde zu einer Datei außerhalb des App-Bundles führen und kann nicht verwendet werden.	MyHybridApp.iOS	C:\Users\hann.HANN-EDV\.nuget\packages\blazorpro.spinkit\1.2.0\contentFiles\any\netstandard2.1\libman.json

If i want to build a Xamarin.IOS on my locally attached iphone i get the error above.
Basically it tells us that the File would be outside the app bundle.
Xamarin.Android buildes just fine!
The application is just pulled from 
https://docs.microsoft.com/en-us/mobile-blazor-bindings/walkthroughs/hybrid-hello-world

Any ideas how to get rid of that or solve the problem?

**Comment by EdCharbeneau** [comment]
Sounds like you're using Maui. Spinkit doesn't support Maui yet as it's still in preview. I'll be waiting to see if the .NET team addresses the issue.

**Comment by whann0205** [comment]
Ed, i am fairly new in the .net/Visual Studio arena. 
So i was mainly trying to find out, why its not working.
I am fully aware that i cannot expect it to work.
And its working with Xamarin.Android perfectly, 
so its probably not so hard to fix it for you.

**Comment by whann0205** [comment]
I love your component and hope i do not have to find an
alternative :-(
How can the .net team get to know this issue 
i have no contact to anyone there.
Its not Maui yet its mobile blazor bindings with .net3.1.

**Comment by EdCharbeneau** [comment]
I posted an update that might help. https://www.nuget.org/packages/BlazorPro.Spinkit/1.1.1-pre
It's in pre-release, so please use the pre-release option in NuGet package manager.

> Its not Maui yet its mobile blazor bindings with .net3.1.

You may still run into issues, these are experimental bits you're using.

I will add support for .NET 6, for use with the Blazor Hybrid template when it is released in November. There are no plans to release a new version of Spinkit until then.

**Comment by whann0205** [comment]
Hi Ed,
I tried with your pre-release but
get again this error:

Der Pfad "..\..\..\blazorpro.spinkit\1.1.1-pre\staticwebassets\spinkit.min.css" würde zu einer Datei außerhalb des App-Bundles führen und kann nicht verwendet werden.	AssetQueryApp.iOS	C:\Users\hann.HANN-EDV\.nuget\packages\blazorpro.spinkit\1.1.1-pre\staticwebassets\spinkit.min.css		

I know i am using experimental things but i have no choice i WANT to use blazor and i have to

**Comment by whann0205** [comment]
make my app work on mobile devices.
net maui isnt ready yet.
Good to hear you will support that, since i will upgrade to maui as soon as its possible.
At the moment i cant even debug with .net preview 4, its buggy as hell.
Thank you very much for trying
